### ISSUE TITLE
Homebrew installation for Movement CLI

### ISSUE THREAD

**Issue by andygolay** [movementlabsxyz/movement#815]

**Comment by Rahat-ch** [comment]
@andygolay would you be able to keep us on devrel in the loop on this as well - we can help test it out and add to docs once ready

**Comment by andygolay** [comment]
> @andygolay would you be able to keep us on devrel in the loop on this as well - we can help test it out and add to docs once ready

Yes, definitely. I'm going to try with this release first https://github.com/movementlabsxyz/aptos-core/releases/tag/movement-cli-v3.5.0
